core/src/transport/and_then.rs

```diff
@@ -21,15 +21,17 @@
 use crate::{
     connection::{ConnectedPoint, Endpoint},
     either::EitherError,
-    transport::{ListenerEvent, Transport, TransportError},
+    transport::{ListenerId, Transport, TransportError, TransportEvent},
 };
 use futures::{future::Either, prelude::*};
 use multiaddr::Multiaddr;
 use std::{error, marker::PhantomPinned, pin::Pin, task::Context, task::Poll};
 
 /// See the `Transport::and_then` method.
```

Comment:
```suggestion
/// See the [`Transport::and_then`] method.
```

Hope you don't mind all the minor unrelated suggestions. Feel free to ignore in case you think it adds to much noise to the pull request.

Reply:
Don't mind them at all. Thanks!